.gitignore

```diff
@@ -139,3 +139,7 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+
+# API test sample
+.test
```

Comment:
no need to add it to .gitignore

Reply:
but it's Elizabeth's test, I use it for reference